app/src/main/java/com/simplecity/amp_library/ui/modelviews/ShuffleAlbumsView.java

```diff
@@ -0,0 +1,18 @@
+package com.simplecity.amp_library.ui.modelviews;
+
+import com.simplecity.amp_library.R;
+import com.simplecity.amp_library.ui.adapters.ViewType;
+
+public class ShuffleAlbumsView extends ShuffleView {
```

Comment:
We could probably get away with just giving the ShuffleView icon drawable a public setter (or a constructor overload to pass in the drawable resource), since the only difference here is the icon being used. It's not a big deal though - I'm glad to see someone using Shuttle's modelviews stuff, seemingly without too much trouble!